### PR TITLE
The even better PR

### PR DESCRIPTION
**Pull request by leogiertz**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/hult/dotfiles/pull/6%23issuecomment-183275538%22%2C%20%22https%3A//github.com/hult/dotfiles/pull/6%23discussion_r52728462%22%2C%20%22https%3A//github.com/hult/dotfiles/pull/6%23discussion_r52728474%22%2C%20%22https%3A//github.com/hult/dotfiles/pull/6%23discussion_r52728527%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hult/dotfiles/pull/6%23issuecomment-183275538%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-02-12T11%3A01%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/374824%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hult%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20be454d045b5b71a824b7d06265168b570e32941c%20.gitignore%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hult/dotfiles/pull/6%23discussion_r52728474%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20is%20an%20abomination%22%2C%20%22created_at%22%3A%20%222016-02-12T11%3A01%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/374824%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hult%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20.gitignore%3AL2-6%22%7D%2C%20%22Pull%20be454d045b5b71a824b7d06265168b570e32941c%20.bash_profile%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hult/dotfiles/pull/6%23discussion_r52728462%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20not%3F%22%2C%20%22created_at%22%3A%20%222016-02-12T11%3A01%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/374824%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hult%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-02-12T11%3A01%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/374824%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hult%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20.bash_profile%3AL6-12%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull be454d045b5b71a824b7d06265168b570e32941c .gitignore 4'></a> &nbsp;<b><a href='https://github.com/hult/dotfiles/pull/6#discussion_r52728474'>File: .gitignore:L2-6</a></b>
- <a href='https://github.com/hult'></a> This is an abomination
- [x] <a href='#crh-comment-Pull be454d045b5b71a824b7d06265168b570e32941c .bash_profile 5'></a> &nbsp;<b><a href='https://github.com/hult/dotfiles/pull/6#discussion_r52728462'>File: .bash_profile:L6-12</a></b>
- <a href='https://github.com/hult'></a> Why not?

<a href='https://www.codereviewhub.com/hult/dotfiles/pull/6?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/hult/dotfiles/pull/6?approve=1'></a>&nbsp;<a href='https://github.com/hult/dotfiles/pull/6'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/hult/dotfiles/pull/6" target="_blank">ReviewNinja

**Comment by hult** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/hult/dotfiles/pull/6'>Refresh</a>.<a href='#crh-update-none'></a>

File: .bash_profile [hunk 1]
```diff
@@ -6,9 +6,6 @@ if [ -f $(brew --prefix)/etc/bash_completion ]; then
   . $(brew --prefix)/etc/bash_completion
 fi
 
-# I hate vim
-export EDITOR=emacs
```
Comment (hult) [inline .bash_profile:10]: Why not?

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@
 *~
 *.pyc
 *.elc
+*
```
Comment (hult) [inline .gitignore:5]: This is an abomination

File: .bash_profile (hunk 1 above)
Comment (hult) [inline .bash_profile:10]: :+1:<a href='#crh-update-complete'></a>

**Comment by hult** [comment]
:+1:<a href='#crh-update-none'></a>

**Comment by hult** [comment]
- [ ] Fixa
- [x] Fixa snabbt
- [ ] Fixa nu!

**Pull request closed without merging**